### DISCUSSION TITLE
Unable to find link to download runtime version 50(.1.1.1)

### DISCUSSION THREAD

**Discussion by wdankier** [airsdk/Adobe-Runtime-Support#2349] [General]
When the releases of the version 50 sdk are available, I assume the runtime must also be available for download.
Because otherwise the sdk 50.x is useless, right?

But for some reason, I am not able to find the runtime for this sdk.
Is there anyone how can point me at my blindspot?

**Comment by ajwfrost** [comment]
We're actually hoping to wean people off the idea of using a separate runtime.. existing .air apps that are out there should work fine with 33.x, and new apps can be deployed via the "bundle" mechanism which keeps the runtime within the package. The native installer mechanisms are going to face increasing problems because of how they're created (particularly on mac with the security aspects that they're bringing in)...

But, we do intend to upload a 50.x runtime release, probably the next 50.1 version..

thanks

**Reply by wdankier** [reply]
Forgot to thank you for your answer.
We can not change our distribution to a "bundle" mechanism. We have a lot of installments with customers. Therefore we pay for a license. We see bug fixes appear that we really would like to test. Customers have a really bad experience because some bugs. We sit on the edge of our chairs to see when we can provide them with an update. But it takes soooooo long for the runtimes to appear.
We pay for a license and we depend on the runtimes. Not really fun to be pushed to the side and have to tell customers every day "sorry" for months...
So I am daily checking for new runtimes. And I do not really understand why it can not be kept up to date (in sync) with the sdk version.

**Reply by ajwfrost** [reply]
Hi

I've asked one of our webserver team to look at updating how this part of the website works, so that we can hook it up with the release database. You're right to a certain extent -> the shared runtimes are built as part of the 'full' release process -> but because of how the website was originally put together, they are not included in the standard release upload process. If we can get that part updated, it will make it more automatic to then keep the runtimes and SDKs in sync.

The 50.1.1.2 runtimes should be up soon, there's a glitch with the Jenkins machine currently.. ("soon" is vague, sorry, I would hope it's today, but not fully within my control!)

thanks

**Reply by ajwfrost** [reply]
FYI: you may need to force-refresh your page, but the 50.1.1.2 runtimes are up now. Apologies for that delay...
